ci-operator/step-registry/openshift/e2e/azure/ovn/upgrade/loki/openshift-e2e-azure-ovn-upgrade-loki-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-e2e-azure-ovn-upgrade-loki
```

Comment:
Should be `openshift-upgrade-e2e-azure-ovn-loki` to match other upgrade workflows